### ISSUE TITLE
pre-trained model request

### ISSUE THREAD

**Issue by playdasegunda** [chomeyama/HN-UnifiedSourceFilterGAN#4]
Hi, is everything okay? I would like to know if it is possible to release the pre-trained model of HN-UnifiedSourceFilterGAN. Would it be possible to make this release for us to test without having to train from scratch? I await your return and thank you in advance.

Att. Lucas.

**Comment by chomeyama** [comment]
Hi, thank you for your interest in our work.
I uploaded a pre-trained model here:
https://www.dropbox.com/s/ustdd79shihi53i/pretrained_hnusfgan.tar.gz?dl=0
Can you get it? 

The hyper parameters and feature extraction settings are writen in the included config file.

**Comment by playdasegunda** [comment]
Thank you very much for the release, I would like to know how I should use the argument for a "test.wav" file,

**Based on the suggested command**: usfgan-decode out_dir=exp/parallel-hn-usfgan/wav/600000steps "C:\Users\lucas\Downloads\HN\pretrained_model\checkpoint-600000steps.pkl"

What is the correct command?

Thank you again, have a great day!

**Comment by chomeyama** [comment]
Sorry for the late reply...
I have added an script for analysis-synthesis (i.e., resynthesis from "test.wav" file).
I think you can use the `usfgan-anasyn` command now. 
The configurations for analysis-synthesis can be customized from `usfgan/bin/config/anasyn.yaml` or commandlines.
Please check it!

**Comment by playdasegunda** [comment]
Hello @chomeyama, thank you for creating this script.

It worked as expected, however I can only process .wav files with a maximum duration of (42 seconds) on a 6 gb vram GPU, is there a "chunks" or "batchsize" option to allow longer file duration? I also noticed that it does not support stereo files, and the maximum supported samplerate is 24000, I tried 44100 but got an error.

I could notice that the UnifiedSourceFilterGAN is not the best option to use when it comes to vocals / acapella,

I found the [SiFIGAN](https://github.com/chomeyama/SiFiGAN) which, when watching the demonstrations, is the best option for "singing"

However, I noticed that there is still no pre-trained model available for it, including I had made
a request on that repository, would it be possible to add the checkpoint, and create the "anasyn.py" option for the SiFIGAN??

Once again, my thanks, it will be very useful and valuable to the music community, cordially, Lucas Rodrigues.
